packages/now-cli/test/integration.js

```diff
@@ -29,8 +29,19 @@ function execa(file, args, options) {
   return _execa(file, args, options);
 }
 
+function fixture(name) {
+  const directory = path.join(__dirname, 'fixtures', 'integration', name);
+  const config = path.join(directory, 'project.json');
+
+  // We need to remove it, otherwise we can't re-use fixtures
+  if (fs.existsSync(config)) {
+    fs.unlinkSync(config);
+  }
```

Comment:
Oh you may have solved another issue I was having with using a fixture in 2 different tests :smart: